pkg/apis/eventing/v1alpha1/trigger_lifecycle.go

```diff
@@ -32,7 +32,7 @@ const (
 
 	TriggerConditionSubscribed apis.ConditionType = "Subscribed"
 
-	TriggerConditionDependencyReady apis.ConditionType = "DependencyReady"
+	TriggerConditionDependency apis.ConditionType = "Dependency"
```

Comment:
Why did this name change? I'm fine with either, I'm just curious what prompted it.

Reply:
Just wanna follow the naming convention of other dependent resources of trigger like Broker.